kubernetes/bin/deploy.sh

```diff
@@ -1,53 +1,59 @@
 #!/bin/bash
 export DIR=kubernetes
 
-USAGE="USAGE: ./kubernetes/bin/deploy-development.sh create|create all|delete|delete all"
+USAGE="USAGE: ./kubernetes/bin/deploy.sh  api|staging create|create all|delete|delete all"
 
-if [[ "$1" = "create" ]];
+if [[ "$1" = "api" ]]
 then
+    export BRANCH=api
+else
+    export BRANCH=staging
+fi
```

Comment:
So that if we want we can change the name of directories if needed.
If `$1` is set as any other random directory name, then setup will just fail with inappropriate errors. I think else condition should be to print error. I will fix this.